source/read-write-concern/read-write-concern.rst

```diff
@@ -301,11 +301,13 @@ Other commands that write
 
 Command helper methods for commands that write, other than those discussed above,
 MAY accept a write concern or write concern options in their parameter list.
+If the helper accepts a write concern, the driver MUST error if the selected server's MaxWireVersion < 5 and a
+write concern has explicitly been specified.
```

Comment:
Should this be specified for `findAndModify` as well? For any driver that hasn't already added an explicit "writeConcern" option but plan to introduce the functionality, they would do well to throw an error if the user specifies the option for MaxWireVersion < 4.